Домашнее задание № 9

leesovska opened the pull request:
Создание таймера обратного отсчета от 25 минут до нуля (Pomodoro).

leesovska committed:
Домашнее задание № 9

Создание таймера обратного отсчета от 25 минут до нуля (Pomidoro).

mdubik777 commented on script.js:
```diff
@@ -0,0 +1,79 @@
+let mode = 'pomodoro';
+
+function startTimer() {
+  let time = timerDisplay.textContent.split(':');
```
Наташа,  почему let? Вроде эту переменную явно не перезаписываем

mdubik777 commented on script.js:
```diff
@@ -0,0 +1,79 @@
+  startButton.textContent = 'start';
+});
+
+modeButton.addEventListener('click', () => {
```
Наташа,  тут получается, что переключение режимов на одной кнопку pomodoro)  Работает корректно,  но раздели плз по кнопкам. Отдельно опиши кнопку pomodoro и отдельно break

mdubik777 commented on script.js:
```diff
@@ -0,0 +1,79 @@
+  stopTimer();
+  resetTimer();
+  startButton.textContent = 'start';
+});
```
А вообще ты большая молодец! Умница!

leesovska committed:
Домашнее задание № 9

Исправлен код для переключения между кнопками 'pomodoro' и 'break'

leesovska commented:
почему-то не срабатывает код, когда таймер доходит до нуля, он должен сброситься в начальное состояние: 25:00 , а у меня 00:00 остается :(

leesovska committed:
Домашнее задание № 9

Исправлена ошибка: при нажатии на break кнопка становится активной.
Также доработан код в части, когда таймер доходит до 00:00, он сбрасывается  в начальное состояние: 25:00/05:00.
